app/assets/stylesheets/application.scss

```diff
@@ -4,3 +4,6 @@
   max-height: 200px;
   background-color: cyan;
 }
+.title {
+  text-align: center;
+}
```

Comment:
Using Foundation, this could be accomplished with the existing `text-center` CSS class. http://foundation.zurb.com/sites/docs/typography-helpers.html#text-alignment